day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,62 @@
+# Create a file named `10_speckled_frogs.js` and within that file, write a program that will print the following nursery rhyme:
+
+# 3 speckled frogs sat on a log
+# eating some most delicious bugs.
+# One jumped in the pool where its nice and cool,
+# then there were 2 speckled frogs.
+#
+# 2 speckled frogs sat on a log
+# eating some most delicious bugs.
+# One jumped in the pool where its nice and cool,
+# then there was 1 speckled frogs.
+
+# 1 speckled frog sat on a log
+# eating some most delicious bugs.
+# One jumped in the pool where its nice and cool,
+# then there were no more speckled frogs!
+
+# Required
+# Make your program print the rhyme above for *10* frogs, with attention to where language changes.
+
+frogs = (1..10).to_a.reverse
```

Comment:
Great work here, Nick! Very cool to see you completing the extension as well. One note that could make things a little easier to read and less repetitive is by storing some of the commonly used copy in variables, and referencing it within your `each` loop for each conditional.